docs/cli-reference/ark_server.md

```diff
@@ -14,7 +14,8 @@ ark server [flags]
 ### Options
 
 ```
-  -h, --help   help for server
+  -h, --help             help for server
+      --log-level enum   the logrus level at which to log (default info)
```

Comment:
Can we spell out the options?

`--log-level <level> the logrus level at which to log (..list of options)` 

I don't `enum` is self explanatory for new users.

Reply:
good idea, will do this.